.travis.yml

```diff
@@ -14,8 +13,9 @@ matrix:
 
 install:
    - go get github.com/mattn/goveralls golang.org/x/tools/cmd/cover github.com/pierrre/gotestcover github.com/Masterminds/glide
-   - glide up
+   - go get -t ./...
```

Comment:
This is why the builds are failing, why don't you remove this? All the dependencies should be installed on glide install